bitrise.yml

```diff
@@ -154,9 +154,6 @@ workflows:
     - fastlane@3:
         inputs:
         - lane: build_and_test
-    - fastlane@3:
-        inputs:
-        - lane: run_xcov
```

Comment:
Based on [this step](https://github.com/nimblehq/ios-templates/blob/master/.github/workflows/automatic_pull_request_review.yml#L50) from the GitHub action workflow, please run `clean_up_xcov` instead and then update the Bitrise document also.

Reply:
@suho I created the issue #296 to cover your comment. Please take a look.